api_spec.yaml

```diff
@@ -2505,6 +2508,58 @@ components:
         tenantId:
             type: string
             example: "customer1"
+        thirdPartyProvider:
+            type: object
+            properties:
```

Comment:
can you reuse thirdPartyProvider schema ?

Reply:
Yes I had already reused the schema wherever we were using it when I created it.

Reply:
why is it not used here? is it different?

Reply:
Because this is the defintion of the schema itself

Reply:
oh right, my bad! in the diff it seemed like something else